docs/plugins/beatport.rst

```diff
@@ -21,15 +23,18 @@ run the :ref:`import-cmd` command after enabling the plugin, it will ask you
 to authorize with Beatport by visiting the site in a browser. On the site
 you will be asked to enter your username and password to authorize beets
 to query the Beatport API. You will then be displayed with a single line of
-text that you should paste into your terminal. This will store the
-authentication data for subsequent runs and you will not be required to
-repeat the above steps.
+text that you should paste as a whole into your terminal. This will store the
+authentication data (under ``~/.config/beatport_token.json``) for subsequent
+runs and you will not be required to repeat the above steps.
```

Comment:
The path to the token file is actually configurable, so I think we should avoid mentioning a hard-coded path here. (And it doesn't seem like most users need to worry about where this is stored?)

Reply:
I agree.